src/EA.Weee.Web/Areas/Admin/Views/ProducerSubmission/Removed.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@model EA.Weee.Web.Areas.Admin.ViewModels.Producers.RemovedViewModel
+@{
+    var title = "Removed producer registration";
+    ViewBag.Title = @title + " - GOV.UK";
+}
+
+@section main_content {
+
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">@title</h1>
+    </header>
+    @using (Html.BeginForm())
+    {
+        @Html.AntiForgeryToken()
+        @Html.HiddenFor(m => m.ProducerName)
+        @Html.HiddenFor(m => m.ComplianceYear)
+
```

Comment:
@stevelee-civica  this might need updating based on the authorised rep conversation.

Reply:
no changes